src/sentry/services/hybrid_cloud/organization/impl.py

```diff
@@ -245,7 +245,7 @@ def set_user_for_organization_member(
 
     def check_organization_by_slug(self, *, slug: str, only_visible: bool) -> Optional[int]:
         try:
-            org = Organization.objects.get_from_cache(slug=slug)
+            org = Organization.objects.get(slug=slug)
```

Comment:
Why no cache here? Seems like a good place to leverage caching.

Reply:
I was debugging an issue, I can return that.